deployment/hooks/custom/das-before-release.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+set -e
+
+release_notes_file_path="$(pwd)/docs/release-notes.md"
+
+function get_repository_changelog() {
+    local changelog_path="$1"
+
+    if verify_file_exists "$changelog_path"; then
+        cat "$changelog_path"
+    fi
+}
+
+function create_release_notes_file() {
+    local release_notes_title="$1"
+    print ":yellow:Creating document at $release_notes_file_path...:/yellow:"
+    mkdir -p "$(dirname "$release_notes_file_path")"
+    echo -e "$release_notes_title\n\n" >"$release_notes_file_path"
+}
+
+function generate_release_notes_block() {
+    local new_block=""
+
+    for package in $(echo "$packages_pending_update" | jq -c '.[]'); do
+        IFS='|' read -r package_name current_version new_version repository_path <<<"$(extract_packages_pending_update_details "$package")"
+
+        new_block+="* $package_name: $new_version\n"
+    done
+
+    echo -e "## DAS Version $new_package_version\n\n$new_block"
+}
+
+function generate_release_notes_changelog_block() {
+    local new_block=""
+
+    for package in $(echo "$packages_pending_update" | jq -c '.[]'); do
+        IFS='|' read -r package_name current_version new_version repository_path repository_name repository_owner repository_workflow repository_ref <<<"$(extract_packages_pending_update_details "$package")"
+
+        local changelog=$(get_repository_changelog "$repository_path/CHANGELOG")
+
+        new_block+="#### $package_name $new_version\n\n\`\`\`\n$changelog\n\`\`\`\n\n"
```

Comment:
It seems like this is not adding the triple quotes "```" at the beginning and the and of the changelog text. This is important because changelogs may contain MARKDOWN syntax characters which will mess with document formatting.

Reply:
These \`\`\` are the triple quotes. I just escaped them in the string.